packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -78,6 +78,24 @@ class CpuProfilePair {
     return CpuProfilePair(functionProfile: function, codeProfile: code);
   }
 
+  factory CpuProfilePair.withTagRoots(
+    CpuProfilePair original,
+    CpuProfilerTagType tag,
```

Comment:
nit: rename to tagType so that it is clear this is a type and not a user / vm tag